lib/formtastic_customizations.rb

```diff
@@ -17,9 +17,15 @@ def hint_html
           
           output = "".html_safe
           output << hint_html_without_customizations
-                    
+
+          current_person = template.try(:current_user).try(:person_type)
+          if current_person.nil?
```

Comment:
I think we'd want to do the `HelpText` selection a little differently here.

Chances are, `current_person` will never be `nil`, but there will be help texts we want to use that aren't scoped to an audience. The code here will essentially ignore any `HelpText`s not scoped by audience.

We could first search for audience scoped texts then search for non-audience scoped texts. But there may be a better/more efficient way, I just can't think of it off the top of my head